lib/strings.js

```diff
@@ -608,7 +608,7 @@ const Strings = new LocalizedStrings({
       noGoBack: '否，返回',
     },
     logout: {
-      popupText: '您确定要退出吗？',
+      popupText: '您確定要刪除您所有儲存下來的資料嗎？',
```

Comment:
@markkhoo if I copy and paste this into Google Translate, it gives me "Are you sure you want to delete all your stored data?" - again, this is in the logout section, so is this the correct change?

Reply:
you are right!